### PR TITLE
Job-seeker Job-seeker-status JS-AP-joint table

### PR DESCRIPTION
**Pull request by amba178**
Need Person model to implement acts_as, commented out JobSeeker model  and spec/job_seeker_spec.rb file
Need AgencyPerson model  for has_and_belongs_to_many test to pass, currently pending in rspec

### CODE REVIEW

File: spec/models/job_seeker_spec.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	it "should not be blank" do 
+		is_expected.to validate_presence_of(:year_of_birth) 
+	end
+
+	it "has many and belongs to agency people"do
+		pending "need to create joint table agency_people<>job_seeker"
+		is_expected.to have_and_belong_to_many :agency_people
+	end
+
+	context "with valid attributes" do
```
Comment (joaopapereira) [inline spec/models/job_seeker_spec.rb]: is 99/99/9999 a valid date?

File: app/models/job_seeker.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class JobSeeker < ActiveRecord::Base
+ 	# acts_as    :person 
+	belongs_to :job_seeker_status
+	has_one    :resume 
+	has_one	   :address, as: :location
+	has_and_belongs_to_many :agency_people, :join_table => "agencies_seekers" 
+
+	validates_presence_of :year_of_birth 
+	validates_format_of :year_of_birth, with: /\A\d{2}\/\d{2}\/\d{4}\z/,
```
Comment (joaopapereira) [inline app/models/job_seeker.rb]: year of birth should be just the year so a 4 digit number or a 4 digit string

File: spec/models/job_seeker_spec.rb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		it {expect(salem).to_not be_valid}
+		it {expect(fatuma).to_not be_valid}
+		it {expect(omar).to_not be_valid}
+	end
+
+	it "is belongs to jobseeeker status,  with valid attributes" do 
+		sam.save
+		@job_seeker_status = JobSeekerStatus.create!(:value => "Employed not looking for job",
+		 :description => "A"*244)
+		# byebug
```
Comment (joaopapereira) [inline spec/models/job_seeker_spec.rb]: Please remove the this comment

File: db/migrate/20151021171711_create_job_seeker_statuses.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateJobSeekerStatuses < ActiveRecord::Migration
+  def change
+    create_table :job_seeker_statuses do |t|
+      t.string :value
+      t.text :description
+
+      t.timestamps null: false
+    end
```
Comment (joaopapereira) [inline db/migrate/20151021171711_create_job_seeker_statuses.rb]: The status should be inserted automatically after this the creation of the table

**Comment by joaopapereira** [comment]
@amba178 please before pushing the changes that I suggested merge the current development branch into yours, or else we will have errors

**Comment by amba178** [comment]
I will do that, Joao.

**Comment by joaopapereira** [comment]
Should this pull request be closed because you opened a new one?

**Comment by amba178** [comment]
Sorry, yes should be closed. Bear with github thing for a week and i will catch up. I promise you.

**Pull request closed without merging**